ansible/roles/nginx/templates/default.j2

```diff
@@ -1 +1,7 @@
 # {{ ansible_managed }}
+
+location /nginx_stub_status {
```

Comment:
Note I haven't actually enabled the nginx monitoring (not sure of the best pattern to use to get apache on one proxy machine and nginx on the other). Any ideas would be helpful.